src/components/Escrow.js

```diff
@@ -1,41 +1,100 @@
 import {useEffect, useState} from 'react';
 
 import getWeb3 from '../web3';
+import { ethers } from 'ethers';
 import EscrowFactoryView from './EscrowFactoryView';
 import EscrowFactoryABI from '../contracts/EscrowFactoryABI.json';
+import EscrowABI from '../contracts/EscrowABI.json';
+import { toast } from 'react-toastify';
 
-export default function EscrowContainer({address, scanner, rpcUrl}) {
-    const [latestEscrow, setLatestEscrow] = useState('');
+export default function EscrowContainer({address, networkMap}) {
     const [count, setCount] = useState(0);
+    const [contractData, setContractData] = useState(['','', '', '', '']);
+    const [scanner, setScanner] = useState("");
+
     const eventsUrl = `${scanner}/address/${address}#events`;
 
+    const notify = () => toast.info('Latest Escrow updated now!', {
+                                    position: "top-center",
+                                    autoClose: 3000,
+                                    hideProgressBar: true,
+                                    closeOnClick: true,
+                                    pauseOnHover: true,
+                                    draggable: true,
+                                    progress: undefined,
+                                });
+    
     useEffect(() => {
         async function setupEscrow() {
             try {
-                const web3 = getWeb3(rpcUrl);
-                const EscrowFactory = new web3.eth.Contract(EscrowFactoryABI, address);
-                const escrowCount = await EscrowFactory.methods.counter().call();
-                setCount(escrowCount);
-    
-                const lastEscrow = await EscrowFactory.methods.lastEscrow().call();
-                setLatestEscrow(lastEscrow);
+                const provider = getWeb3(networkMap['polygon'].rpcUrl);
+                const EscrowFactory = new ethers.Contract(networkMap['polygon'].defaultFactoryAddr, EscrowFactoryABI, provider)
+                let escrowCount = await EscrowFactory.counter();
+                let lastEscrow = await EscrowFactory.lastEscrow();
+                let eip = await EscrowFactory.eip20();
+                let escrowCounters = await EscrowFactory.escrowCounters(lastEscrow);
+                let Escrow = new ethers.Contract(lastEscrow, EscrowABI, provider)
+                let balance = await Escrow.getBalance();
+                setScanner('https://polygonscan.com');
```

Comment:
you should get this value from the configuration(or constants) instead of hardcoding it